python/python/tests/test_async_client.py

```diff
@@ -3326,10 +3328,25 @@ async def test_xadd_xtrim(self, redis_client: TRedisClient):
             )
             is not None
         )
-        assert await redis_client.custom_command(["XLEN", key]) == 2
+        assert await redis_client.xlen(key) == 2
 
         assert await redis_client.xtrim(key, TrimByMaxLen(threshold=1, exact=True)) == 1
-        assert await redis_client.custom_command(["XLEN", key]) == 1
+        assert await redis_client.xlen(key) == 1
+
```

Comment:
With streams you can remove all data from a stream and it will make it empty, but key remains exist (dislike other collection types in redis). Please test such case.

Reply:
Added now